src/tasks/fork_repo.rs

```diff
@@ -4,7 +4,6 @@ use tracing_batteries::prelude::*;
 
 pub struct ForkRepository {
```

Comment:
With this just basically delegating to the `fork_remote` functionality, should we call it `ForkRemote`?